src/www/ui/api/Controllers/UploadController.php

```diff
@@ -1093,7 +1093,13 @@ public function getAllAgents($request, $response, $args)
         $item['currentAgentRev'] = "";
       }
       foreach ($item['successfulAgents'] as &$agent) {
-        $agent['agent_id'] = intval($agent['agent_id']);
+        $agent['agentId'] = intval($agent['agent_id']);
+        $agent['agentRev'] = $agent['agent_rev'];
+        $agent['agentName'] = $agent['agent_name'];
+        unset($agent['agent_id']);
+        unset($agent['agent_name']);
+        unset($agent['agent_rev']);
```

Comment:
I would instead recommend to create a new model and get the version aware array from it. See https://github.com/fossology/fossology/pull/2617/files#diff-f62f1d2b860c504c87d386cdfa79513c109848593f0afeb60a5dc76e4b98ff42 for reference.